mlflow/server/js/src/utils/Utils.js

```diff
@@ -388,6 +388,13 @@ class Utils {
       !t[0].startsWith(MLFLOW_INTERNAL_PREFIX)
     );
   }
+
+  static getAjaxUrl(relativeUrl) {
+    if (process.env.USE_ABSOLUTE_AJAX_URLS && process.env.USE_ABSOLUTE_AJAX_URLS === "true") {
```

Comment:
Do we need this extra check `process.env.USE_ABSOLUTE_AJAX_URLS` here?  Is `process.env.USE_ABSOLUTE_AJAX_URLS === "true"` sufficient?

Reply:
Oops yeah you're right, I'll get rid of the redundant check :p - thanks!